src/layouts/createStartPageMain.ts

```diff
@@ -0,0 +1,22 @@
+import getHtmlElement from '../components/getHtmlElement';
+import getLangObj from '../features/getLangObj';
+
+function createStartPageMain() {
+    const langObj = getLangObj();
+
+    document.querySelector('.main')?.remove();
+
+    getHtmlElement({ parent: 'body', tag: 'main', style: ['main', 'main_start-page'] });
```

Comment:
This function `get` element or `set/add/draw` it into parent tag?

Reply:
But what if the function is universal, it both creates an element and returns it? Sometimes it is necessary.

Reply:
I don't see this function. All ok, if u'r function is universal.